pkgs/by-name/gi/git-lfs-transfer/package.nix

```diff
@@ -1,20 +1,25 @@
 {
   lib,
+  go,
   buildGoModule,
   fetchFromGitHub,
 }:
 buildGoModule {
   pname = "git-lfs-transfer";
-  version = "0.1.0-unstable-2024-10-07";
+  version = "0.1.0-unstable-2025-05-05";
 
   src = fetchFromGitHub {
     owner = "charmbracelet";
     repo = "git-lfs-transfer";
-    rev = "422d24414fe4b803849b3f6fe7c4d8ab1b40803b";
-    hash = "sha256-YsplPW3i4W1RfkWQI1eGXFXb3JofQwKe+9LbjxeL1cM=";
+    rev = "0d51139d5979491e819ba4751235e2ab4b763c6e";
+    hash = "sha256-cufY0CdkTIVHFH3LesAxMDV4KNIqjxFO2m3dLoBNM9U=";
   };
+  postPatch = ''
+    substituteInPlace go.mod \
+      --replace-fail "go 1.21" "go ${go.version}"
+  '';
```

Comment:
```suggestion
```
Why do you change that? it will work without it

Reply:
You are right, `postPatch` is not needed to build this package.
When preparing this commit, I set `vendorHash` to b.fakeHash` to get the correct `vendorHash`. The build system asked me to run `go mod tidy` and did not give me the right hash. Only with this `postPatch`, I could get the correct `vendorHash`. But after that, it is no longer needed.